aiohttp/web_protocol.py

```diff
@@ -336,13 +337,15 @@ def close(self):
             if not waiter.done():
                 waiter.cancel()
 
-    def force_close(self):
+    def force_close(self, keepalive=False):
```

Comment:
I think the argment name a bit misleading. `send_last_heartbeat` - might be more precisely about what will happen. The `keepalive` fits more to class property which can be enabled or disabled.